medusa/helpers.py

```diff
@@ -716,7 +716,7 @@ def sanitize_scene_name(name, anime=False):
         name = name.replace(x, '')
 
     # tidy up stuff that doesn't belong in scene names
-    name = name.replace('- ', '.').replace(' ', '.').replace('&', 'and').replace('/', '.')
+    name = name.replace('- ', '.').replace(' ', '.').replace('&', 'and').replace('/', '.').replace(': ', '-')
```

Comment:
Maybe for SCENE releases, but that doesn't mean all releases are named that way.  Also most sites are keyword searches so it *should* be safer to use ' ' over '-'.